chainstate/src/detail/error.rs

```diff
@@ -165,7 +172,7 @@ pub enum InitializationError {
     #[error("Block storage error: `{0}`")]
     StorageError(#[from] chainstate_storage::Error),
     #[error("{0}")]
-    PropertyQuery(#[from] PropertyQueryError),
+    PropertyQueryError(#[from] PropertyQueryError),
```

Comment:
This variant is actually never used. Should I delete it instead?

Reply:
Yep

Reply:
Oops, I lied, it is used, just not explicitly.